### PR TITLE
Fix internal field in IGenerateAddressesOptions

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Fix internal field in IGenerateAddressesOptions

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Running an example with the internal field set to true

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 1.0.0-rc.1 - 2023-06-19
 
+### Fixed
+ - Moved `internal` field from `IGenerateAddressesOptions` to `IGenerateAddressOptions`;
+
+## 1.0.0-rc.1 - 2023-06-19
```
Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md:28]: Something got messed up a bit here

File: bindings/nodejs/examples/secret_manager/generate-addresses.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ async function run() {
                 start: 0,
                 end: 4,
             },
-            internal: true,
+            options: { internal: true },
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/secret_manager/generate-addresses.ts:51]: Not needed in the old bindings?

Comment (Thoralf-M) [reply bindings/nodejs/examples/secret_manager/generate-addresses.ts:51]: The old binding is not needed :trollface: 
Will update

File: bindings/nodejs/CHANGELOG.md (hunk 1 above)
Comment (Thoralf-M) [reply bindings/nodejs/CHANGELOG.md:28]: Forgot to update the version and date, updated now

File: bindings/nodejs/CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.0-rc.2 - 2023-0x-xx
+
+### Fixed
+ - Moved `internal` field from `IGenerateAddressesOptions` to `IGenerateAddressOptions`;
```
Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion

  - Moved `internal` field from `IGenerateAddressesOptions` to `IGenerateAddressOptions`;
```

File: bindings/nodejs/CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.0-rc.2 - 2023-0x-xx
+
+### Fixed
+
+  - Moved `internal` field from `IGenerateAddressesOptions` to `IGenerateAddressOptions`;
```
Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
- Moved `internal` field from `IGenerateAddressesOptions` to `IGenerateAddressOptions`;
```

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**